src/v0/destinations/active_campaign/util.js

```diff
@@ -1,31 +1,25 @@
 const { NetworkError } = require('@rudderstack/integrations-lib');
 const {
-  nodeSysErrorToStatus,
   getDynamicErrorType,
+  processAxiosResponse,
 } = require('../../../adapters/utils/networkUtils');
 const tags = require('../../util/tags');
 
 const errorHandler = (err, message) => {
```

Comment:
The name `err` itself is not self-explanatory in that it is a response from Axios lib as we're processing it in the next step.

Reply:
Are you saying to replace `err` with something more meaningful ? or Is it something else ?

Reply:
Yes, rename it to a better name.

Reply:
Done